processes/security_team_membership_policy.md

```diff
@@ -0,0 +1,28 @@
+# Node.js Security Team Membership Policy
+
+The Node.js Security Team (@nodejs/security) has
+access to security-sensitive issues and patches that aren't approriate for
+public availability.
+
+> Note: Due to implementation details and a bit of human error, the list of
+> people with access to the security patches isn't 100% in line with the
+> @nodejs/security membership. See the [membership list](./security_team_members.md)
+> for details. This situation is intended to be corrected in the near future.
+
+This isn't to be confused with the security triage team, which is currently
+defined by the [`security@nodejs.org` email alias](https://github.com/nodejs/email/blob/master/iojs.org/aliases.json).
+Inclusion in that group is on a volunteer basis, upon approval by the Technical
+Steering Committee (ie. @nodejs/TSC).
+
+The policy for inclusion on the @nodejs/security is as follows:
+
+1. All members of the Technical Steering Committee (i.e. @nodejs/TSC) are
+   members of @nodejs/security.
+2. On a case-by-case basis, individuals outside the Technical Steering Committee
+   are invited by the TSC to join @nodejs/security so that their expertise can
+   be applied to an issue or patch. Once their assistance is no longer required
+   for a particular issue or patch, they are removed from the team.
+3. Members of the [release team](https://github.com/nodejs/node#release-team)
+   who are not also members of the TSC are also members of @nodejs/security, as
```

Comment:
Nit: remove `who are not also members of the TSC`

Reply:
+1

As all TSC members currently have access

This may result in us wanting to audit and remove inactive release team memebrs

I think we may want to have a conversation in @nodejs/release to determine if we want the releasers team at large added

Reply:
Discussion would be good.  At a minimum I think we should be saying that any member of the release team that will do a security release can be added, and that provided they have done one in the last year they can stay. That will make it easier to co-ordinate.  I'm also ok with all of the Release team members being added, provided we trim it down to active members.